protofsm/state_machine.go

```diff
@@ -101,8 +102,8 @@ type DaemonAdapters interface {
 	// TODO(roasbeef): could abstract further?
 	RegisterConfirmationsNtfn(txid *chainhash.Hash, pkScript []byte,
 		numConfs, heightHint uint32,
-		opts ...chainntnfs.NotifierOption,
```

Comment:
Prob the main stylistic diff in the codebase rn, but I prefer this option as it makes it clearer where the args end, and the return value starts.

Reply:
Imo that only holds if the return values can in-fact fit on one line which is not always the case. 
Personally I find the `) {` on the unindented line strange